Fhi.HelseId/Api/Authorization/CustomAuthenticationHandler.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+
+
+namespace Fhi.HelseId.Api.Authorization
+{
+    /// <summary>
+    /// This is taken from package: Microsoft.AspNetCore.Authentication.JwtBearer, class JwtBearerHandler.
+    /// See the code by going into the definition, or this link: https://github.com/dotnet/aspnetcore/blob/99f2ba5fd1d668fecbff1e1a9ba6a541c99bec79/src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs#L15
+    /// !IMPORTANT! the only thing we to is to add DPoP for authorazation
+    /// </summary>
+    public class CustomAuthenticationHandler(
```

Comment:
Burde vi kalle denne klassen noe annet for å indikere at det er en klasse som legger til støtte for dpop-tokens?